test/e2e_node/services/services.go

```diff
@@ -211,8 +211,10 @@ func (e *E2EServices) startKubelet() (*server, error) {
 		"--eviction-pressure-transition-period", "30s",
 		"--feature-gates", framework.TestContext.FeatureGates,
 		"--v", LOG_VERBOSITY_LEVEL, "--logtostderr",
-		"--experimental-mounter-path", framework.TestContext.MounterPath,
-		"--experimental-mounter-rootfs-path", framework.TestContext.MounterRootfsPath,
+
+		// Temporarily disabled:
```

Comment:
Not blocking this PR, just a nit: it'd be nice to add a link to a github issue to explain the reasons and track when we can re-enable it.